docker/launch-netbox.sh

```diff
@@ -22,27 +47,37 @@ load_configuration() {
   # this curl call will get a reply once unit is fully launched
   curl --silent --output /dev/null --request GET --unix-socket $UNIT_SOCKET http://localhost/
 
-  echo "⚙️ Applying configuration from $UNIT_CONFIG"
+  if [[ -n "$UNIT_CONFIG" ]] && [[ -s "$UNIT_CONFIG" ]]; then
+    echo "⚠️ The UNIT_CONFIG environment variable is deprecated. All *.pem and *.json files in /etc/unit will be loaded automatically when UNIT_CONFIG is undefined."
+    echo "⚙️ Applying configuration from UNIT_CONFIG environment variable: $UNIT_CONFIG"
+    put_config $UNIT_CONFIG "config" || return 1
+  else
+    echo "🔍 Looking for certificate bundles in /etc/unit/..."
+    for f in $(find /etc/unit/ -type f -name "*.pem"); do
+      echo "⚙️ Uploading certificates bundle: $f"
+      put_config $f "certificates/$(basename $f .pem)" || return 1
+    done
```

Comment:
```suggestion
    for pem_file in /etc/unit/*.pem; do
      echo "⚙️ Uploading certificates bundle '$pem_file'"
      put_config $f "certificates/${pem_file%.json}" || return 1
    done
```

I believe the invocations of `find` and `baseline` should not be necessary.